reV/handlers/multi_year.py

```diff
@@ -126,15 +128,53 @@ def _copy_dset(self, source_h5, dset, meta=None, pass_through=False):
             self._create_dset(dset_out, ds_data.shape, ds_dtype,
                               chunks=ds_chunks, attrs=ds_attrs, data=ds_data)
 
+    @staticmethod
+    def parse_source_files_pattern(source_files):
+        """Parse a source_files pattern that can be either an explicit list of
```

Comment:
So in a lot of places in reV we violate PEP 257:

"Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line." 
(https://peps.python.org/pep-0257/#:~:text=Multi%2Dline%20docstrings%20consist,line%20(see%20example%20below).)

Was just wondering if there was good reason to? Are we just attempting to stick with precedent?

Reply:
Ahh yeah that's probably just an oversight. I don't think i've ever run into a problem with multi-line docstrings so i got lazy with that pep. It looks fine in [our documentation](https://nrel.github.io/reV/_autosummary/reV.bespoke.bespoke.BespokeSinglePlant.html#reV.bespoke.bespoke.BespokeSinglePlant) which is really what i'd be worried about. What do you think? Important enough to change?

Reply:
Nah, I think we should leave it as is, I was mostly just curious. Thanks!

Reply:
Agreed, maybe we can try and be better about this with new code but i dont think it's worth spending the time to go fix everywhere